typings/web3js-quorum/index.d.ts

```diff
@@ -0,0 +1,517 @@
+declare module "web3js-quorum" {
```

Comment:
@zondervancalvez This file looks like it's a copy paste of https://github.com/ConsenSys/web3js-quorum/blob/master/src/typescript/index.d.ts
Why did you need to copy it here instead of using the types directly from the node_modules folder?

Reply:
This screenshot shows that we tried to use the types directly from the node_modules folder:
![index d ts problem (1)](https://user-images.githubusercontent.com/69126996/158170777-96c81bcb-80f8-43d1-a7cb-2f2dbe420b98.jpg)

This one, we tried to apply the suggested fix from vscode. Still getting an error.
![npm -i savedev web3jsquorum error](https://user-images.githubusercontent.com/69126996/158170782-7ec1d16b-e3c1-4f19-871a-04340603e2ba.JPG)

That's why we come up to a solution to use the index.d.ts of web3js-quorum. This method also applies with the existing web3-eea.

Reply:
@zondervancalvez Oh, my bad. I opened an issue [1] for the release to be pushed to npm with the typings. Once that's done we can remove this file from our repo so that we have less code to maintain. For now, LGTM

[1] https://github.com/ConsenSys/web3js-quorum/issues/45